src/cbc_sdk/platform/devices.py

```diff
@@ -172,21 +171,11 @@ def update_sensor_version(self, sensor_version):
         """
         return self._cb.device_update_sensor_version([self._model_unique_id], sensor_version)
 
-    def vulnerability_refresh(self, vcenter_specific=False):
```

Comment:
So we do not need to cover this API: https://developer.carbonblack.com/reference/carbon-black-cloud/workload-protection/latest/vulnerability-assessment/#perform-action-on-a-vcenter-device ?

Reply:
The problem with this API endpoint is it requires the vm_id which is not available in the Device model so we are unable to perform this request. I've also found that there are no vcenter devices that don't have a device id so this API endpoint is mainly for the vSphere application to make API calls through the Appliance VM